docs/ref/databases.txt

```diff
@@ -146,6 +146,10 @@ password from the `password file`_, you must specify them in the
         }
     }
 
+.. note::
+
+    The PostgreSQL backend passes the content of :setting:`OPTIONS` as keyword arguments to the connection constructor, allowing for more advanced control of driver behavior. Any options from the :setting:`OPTIONS` key that are not recognized by the engine itself are passed directly to the underlying ``libpq`` library. This way you can set options like `sslmode`_.
```

Comment:
IMO, it would be useful to link to the list of available parameters:

https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-PARAMKEYWORDS